src/predicates/propPathEq.js

```diff
@@ -0,0 +1,35 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Karthik Iyengar (karthikiyengar) */
+
+const curry = require('../core/curry')
+const isArray = require('../core/isArray')
+const isInteger = require('../core/isInteger')
+const isNil= require('../core/isNil')
+const isString = require('../core/isString')
+const equals = require('../core/equals')
+const Maybe = require('../core/Maybe')
+
+const { Just, Nothing } = Maybe
+
+const lift = x => (!isNil(x) ? Just(x) : Nothing())
+
+// propPathEq :: [String | Number] -> a -> Object -> Boolean
+function propPathEq(path, key, target) {
```

Comment:
To avoid 🌽fusion, `key` should be renamed to something like `value` or something better if you have a better name.